src/PAModel/EditorState/EditorStateFile.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+
+namespace Microsoft.PowerPlatform.Formulas.Tools.EditorState
+{
+    /// <summary>
+    /// Represents the `editorstate.json` file created to contain ControlStates,
+    /// which have studio state content that don't wind up in the IR.
+    /// </summary>
+    internal class EditorStateFile
```

Comment:
Naming nit: maybe `TopParentTreeState`? It's not a file, the file ending could be confusing

Reply:
Agreed on the "File". I don't want to misrepresent this object as the top parent of the tree (since that control is actually contained in `ControlStates`). Thoughts on `EditorControlState`? If it still seems unclear I'm fine to go with `TopParentTreeState` as well.

Reply:
I went with `ControlTreeState`. Let me know what you think and I'll adjust if needed.